### PR TITLE
feat: new README

### PR DESCRIPTION
**Pull request by bollu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,34 +1,65 @@
 # SSA
```
Comment (goens) [inline README.md:1]: would be nice to have more context about what the project is

File: README.md [hunk 2]
```diff
@@ -1,34 +1,65 @@
...
 
+#### Installation
+
+- First, setup the Lean toolchain with [elan](https://github.com/leanprover/elan?tab=readme-ov-file#installation).
+- Next, run:
+
+```
+git clone https://github.com/opencompl/ssa.git && cd ssa && lake exe cache get && lake build
+```
+
```
Comment (goens) [inline README.md:19]: should we have some text about how to prove rewrites correct, etc?

**Review by goens — APPROVED** [review]
changes look good to me, not sure the readme is ready yet though

File: README.md (hunk 2 above)
Comment (bollu) [reply README.md:19]: @goens I'd suggest we do this in a separate PR where we point people to the PaperExamples subdirectory, perhaps?

File: artifact-evaluation/artifact.tex [hunk 3]
```diff
@@ -124,31 +124,39 @@ \subsection{Evaluation and expected results}
...
-% $ cd /code/lz/test/lambdapure/compile/bench && \
-% ./speedup-rgn-time.py --data --plot --nruns 10
-% \end{minted}
-
+\begin{minted}[fontsize=\tiny]{text}
+$ cd /code/ssa
+$ lake build # builds successfully.
+$ cd test/bruteforce-correctness && ./run.sh # brute forcs and runs consistency check with alive semantics upto bitwidth < 4.
++ rm 'generated*'
+rm: cannot remove 'generated*': No such file or directory
```
[the remark this answers is not in the document]
Comment (goens) [reply artifact-evaluation/artifact.tex:132]: `rm -fr`?

File: README.md (hunk 1 above)
Comment (bollu) [reply README.md:1]: @goens got a good summary? (:

File: artifact-evaluation/artifact.tex (hunk 3 above)
[the remark this answers is not in the document]
Comment (bollu) [reply artifact-evaluation/artifact.tex:132]: Resolved.

**Comment by bollu** [comment]
@goens I'm choosing to merge this, and iterate on the two requests for:

1. A more extensive README, and 
2. A deeper example for how to use the library

in subsequent PRs. We can take inspiration from https://github.com/mit-plv/koika

**Pull request merged**